superset/assets/javascripts/dashboard/components/GridLayout.jsx

```diff
@@ -14,117 +12,94 @@ const ResponsiveReactGridLayout = WidthProvider(Responsive);
 
 const propTypes = {
   dashboard: PropTypes.object.isRequired,
+  datasources: PropTypes.object,
+  charts: PropTypes.object.isRequired,
+  filters: PropTypes.object,
+  timeout: PropTypes.number,
+  onChange: PropTypes.func,
+  getFormDataExtra: PropTypes.func,
+  fetchSlice: PropTypes.func,
```

Comment:
you should have defaults for all of these non-required funcs / they would throw below

Reply:
fixed.